app/services/analytics_events.rb

```diff
@@ -2660,13 +2660,14 @@ def multi_factor_auth_added_phone(enabled_mfa_methods_count:, **extra)
     )
   end
 
+  # @identity.idp.previous_event_name Multi-Factor Authentication: Added PIV_CAC
   # Tracks when the user has added the MFA method piv_cac to their account
   # @param [Integer] enabled_mfa_methods_count number of registered mfa methods for the user
   # @param [Boolean] in_account_creation_flow whether user is going through creation flow
   def multi_factor_auth_added_piv_cac(enabled_mfa_methods_count:, in_account_creation_flow:,
                                       **extra)
     track_event(
-      'Multi-Factor Authentication: Added PIV_CAC',
+      'Multi-Factor Authentication: Added PIV CAC',
```

Comment:
This change could break some Cloudwatch queries for historical data. For example, querying for the updated "PIV CAC" will not return events for 'Multi-Factor Authentication: Added PIV_CAC' depending on how the query is written.

For this reason I would revert the change, as I otherwise don't see the gain.